.github/workflows/buildimages-update.yml

```diff
@@ -73,6 +72,11 @@ jobs:
         id: current_go_version
         run: |
           echo "GO_VERSION=$(inv go-version)" >> $GITHUB_OUTPUT
+    
+      - name: Get current buildimage tag
+        id: current_buildimage_tag
+        run: |
+            echo "BUILDIMAGE_TAG=$(inv pipeline.get-image-tag)" >> $GITHUB_OUTPUT
```

Comment:
Do we need to run this step to store the image_tag only to later be use in one place?

Could we inline this in the  `Create PR` step?

Reply:
This step fetches the current buildimage Id, and the next step updates it, we have to query it before it's updated

Reply:
Thank for the explanation

Reply:
I think you should not rely on the version defined in circle CI config file. We plan to get rid of circle CI and it would probably better to fetch this version in `.gitlab-ci.yml`

Reply:
I guess the issue is that `.gitlab-ci.yml` contains several entries which could be different...